tests_e2e/tests/publish_hostname/publish_hostname.py

```diff
@@ -55,7 +55,7 @@ def add_vm_password(self):
 
     def check_and_install_dns_tools(self):
         lookup_cmd = "dig -x {0}".format(self._private_ip)
-        dns_regex = r"[\S\s]*;; ANSWER SECTION:\s.*PTR\s*(?P<hostname>.*).internal.cloudapp.net.[\S\s]*"
+        dns_regex = r"[\S\s]*;; ANSWER SECTION:\s.*PTR\s*(?P<hostname>.*).internal.*[\S\s]*"
```

Comment:
should we use "internal\.(cloudapp\.net|chinacloudapp\.cn|usgovcloudapp.net)" instead?